Makefile

```diff
@@ -141,14 +145,16 @@ $(PROTO_MODEL): data/unicharset data/radical-stroke.txt
 	  --output_dir data \
 	  --lang $(MODEL_NAME)
 
+#      19 [1,48,0,1Ct3,3,16Mp3,3Lfys64Lfx96Lrx96Lfx512O1c1]
+
 ifdef START_MODEL
 $(LAST_CHECKPOINT): unicharset lists $(PROTO_MODEL)
 	mkdir -p data/checkpoints
 	lstmtraining \
 	  --traineddata $(PROTO_MODEL) \
 	  --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
 	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
-	  --net_spec "[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c`head -n1 data/unicharset`]" \
+	  --net_spec "$(subst c1,c`head -n1 data/unicharset`,$(NET_SPEC))" \
```

Comment:
Not completely sure whether this save under all possible configurations (i.e. is it possible that `c1` may be part of the netspec?).